ingest/ingest-datasets-refactored.py

```diff
@@ -0,0 +1,72 @@
+import functools
+import argparse
+import warnings
+import pprint
+
+
+# Import helper functions from ingestHelpersRefactored.py;
+# see ingestHelpersRefactored.py for details of functions and classes used.
+from ingestHelpersRefactored import *
+
+# Directions:
+
+# First, change these case-varying variables below for: dataset ingest
+
+GET_OBJECTS_QUERY_LOCATION = "queries/listDatasets.rq"
+DESCRIBE_DATASET_OBJECT_LOCATION = "queries/describeDataset.rq"
+INDEX = "dco"
+TYPE = "dataset"
+VARIABLE_NAME_SPARQL = "?dataset"
+
+# Second, change the function name "create_xxxxx_doc" below to whatever fits this ingest process
+#       and then implement it in ingestHelpersRefactored.py. Existing examples are helpful.
+if __name__ == "__main__":
+    Main(get_objects_query_location=GET_OBJECTS_QUERY_LOCATION,
+         describe_object_query_location=DESCRIBE_DATASET_OBJECT_LOCATION,
+         create_object_doc_function=create_dataset_doc,  # Need to create case-varying create_xxxxxx_doc in helper file
+         object_index=INDEX, object_type=TYPE,
+         variable_name_sparql=VARIABLE_NAME_SPARQL)
+
+# Finally, run the ingest process. See explanations and examples below.
+
+# Usage:
```

Comment:
Oh ... I see. These comments will be replicated then between the different ingest scripts. Wondering if we should add these to a README file in this directory instead of in code. Or add it as part of the main README.md under an ingest section? @zednis what do you think?